Policies/Kubernetes/allowed-host-paths/rego/src_test.rego

```diff
@@ -0,0 +1,243 @@
+    results := violation with input as input
+    count(results) == 1
+}
+test_input_hostpath_allowed_is_writable {
+    input := { "review": input_review, "parameters": input_parameters_in_list_writable}
+    results := violation with input as input
+    count(results) == 0
+}
+test_input_hostpath_not_allowed_is_writable {
+    input := { "review": input_review, "parameters": input_parameters_not_in_list_writable}
+    results := violation with input as input
+    count(results) == 1
+}
+test_input_hostpath_not_allowed {
+    input := { "review": input_review, "parameters": input_parameters_not_in_list_writable}
+    results := violation with input as input
+    count(results) == 1
+}
+test_input_hostpath_allowed_readonly {
+    input := { "review": input_review_many, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 0
+}
+
+# Init Containers
+test_input_hostpath_allowed_readonly_init_containers {
+    input := { "review": input_init_review, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
+test_input_hostpath_allowed_readonly_many_init_containers {
+    input := { "review": input_init_review_many, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 0
+}
+test_input_hostpath_not_allowed_readonly_init_containers {
+    input := { "review": input_init_review, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
+test_input_hostpath_many_not_allowed_readonly_init_containers {
+    input := { "review": input_init_review_many, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) == 2
+}
+
+input_review = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "containers": input_containers_one,
+            "volumes": input_volumes
+      }
+    }
+}
+
+input_init_review = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "initContainers": input_containers_one,
+            "volumes": input_volumes
+      }
+    }
+}
+
+input_review_writable = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "containers": input_containers_writable,
+            "volumes": input_volumes
+      }
+    }
+}
+
+input_review_many = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "containers": input_containers_many,
+            "volumes": input_volumes_many
+      }
+    }
+}
+
+input_init_review_many = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "initContainers": input_containers_many,
+            "volumes": input_volumes_many
+      }
+    }
+}
+
```

Comment:
Can you please add tests for below input review types...
1. input review with just containers and no volumes
1. input review with containers and volumes but volumes do not have host path in it.